### PR TITLE
Remove regionalization

### PR DESCRIPTION
**Pull request by Kiel-SFU**

### CODE REVIEW

**Review by trevorb1 — CHANGES_REQUESTED** [review]
Functionality, seemed good ! Couple of notes though: 
1. I think we should be consistent with what we are calling regions/subregions/countries (see issue #81). This was my fault from when we first wrote everything. Nothing needed for this pr, but I bumped up priority in the project board to address this issue 
2. The snakefile still references `Rationalization.xlsx` a lot. These references need to be removed
3. The  `Rationalization.xlsx` file in `/data` needs to be removed

File: scripts/RETags.py [hunk 1]
```diff
@@ -29,39 +29,15 @@ def main():
     #print all values 
     for region in regions:
         for year in years:
-            for subregion in subregions:
-                for fuel in fuels:
-                    techName = 'PWR' + fuel + 'CAN' + subregion + '01'
```
Comment (Kiel-SFU) [inline scripts/RETags.py:34]: The removal of 'CAN' here is a miss on my part, glad you caught it

File: scripts/functions.py [hunk 2]
```diff
@@ -137,170 +137,166 @@ def daylightSavings(inData):
 
     return inData
 
-def getPWRtechs(regions, techs):
+def getPWRtechs(region, techs):
     # PURPOSE: Creates all the PWR naming technologies
-    # INPUT:   regions =  Dictionary holding region as the key and subregion as the values in a list
+    # INPUT:   regions =  Tuple holding country as the key and subregion as the values in a dictionary
+    #                     (CAN, {WS:[...], MW:[]...},)
```
Comment (Kiel-SFU) [inline scripts/functions.py:143]: Nitpick: You changed regions -> region in the function parameters but region -> regions in the function description! The same typo is in all of the similar functions.

File: scripts/functions.py [hunk 3]
```diff
@@ -312,48 +308,50 @@ def createFuelDataframe(subregions, rnwFuels, mineFuels):
...
+        minList = getMINtechs(country, mineFuels)
 
         # get Renewables fuels list 
-        rnwList = getRNWtechs(subregions[i], rnwFuels)
-
-        # get trade technology list 
-        trnList = getTRNtechs(trnTechsCsvPaths[i])
+        rnwList = getRNWtechs(country, rnwFuels)
 
         #Append lists together and write to a dataframe
```
Comment (Kiel-SFU) [inline scripts/functions.py:344]: "and write to a dataframe" is no longer accurate (my miss from earlier)

**Review by Kiel-SFU — COMMENTED** [review]
Some very small comments before pulling, but it looks and works great on my end.

**Pull request merged**